src/test/java/reposense/parser/ArgsParserTest.java

```diff
@@ -65,12 +65,14 @@ public void parse_allCorrectInputs_success() throws ParseException, IOException
 
         List<String> expectedFormats = Arrays.asList("java", "adoc", "html", "css", "js");
         Assert.assertEquals(expectedFormats, cliArguments.getFormats());
+
```

Comment:
Unnecessary line break. Also, add an assertion to check that the `isStandaloneConfigIgnored` is `true`.